Makefile

```diff
@@ -81,14 +82,20 @@ generate-crds: ## Generate CRDs and Go types using kubebuilder
 	go run sigs.k8s.io/controller-tools/cmd/controller-gen object paths=./apis/...
 
 .PHONY: generate-manifests
-generate-manifests: ## Generate manifests using Helm.
+generate-manifests: generate-manifests-plus ## Generate manifests using Helm.
 	cp $(CHART_DIR)/crds/* $(MANIFEST_DIR)/crds/
 	helm template nginx-gateway $(CHART_DIR) $(HELM_TEMPLATE_COMMON_ARGS) $(HELM_TEMPLATE_EXTRA_ARGS_FOR_ALL_MANIFESTS_FILE) -n nginx-gateway | cat $(strip $(MANIFEST_DIR))/namespace.yaml - > $(strip $(MANIFEST_DIR))/nginx-gateway.yaml
 	helm template nginx-gateway $(CHART_DIR) $(HELM_TEMPLATE_COMMON_ARGS) --set metrics.enable=false -n nginx-gateway -s templates/deployment.yaml > conformance/provisioner/static-deployment.yaml
 	helm template nginx-gateway $(CHART_DIR) $(HELM_TEMPLATE_COMMON_ARGS) -n nginx-gateway -s templates/service.yaml > $(strip $(MANIFEST_DIR))/service/loadbalancer.yaml
 	helm template nginx-gateway $(CHART_DIR) $(HELM_TEMPLATE_COMMON_ARGS) --set service.annotations.'service\.beta\.kubernetes\.io\/aws-load-balancer-type'="nlb" -n nginx-gateway -s templates/service.yaml > $(strip $(MANIFEST_DIR))/service/loadbalancer-aws-nlb.yaml
 	helm template nginx-gateway $(CHART_DIR) $(HELM_TEMPLATE_COMMON_ARGS) --set service.type=NodePort --set service.externalTrafficPolicy="" -n nginx-gateway -s templates/service.yaml > $(strip $(MANIFEST_DIR))/service/nodeport.yaml
 
+.PHONY: generate-manifests-plus
+generate-manifests-plus: ## Generate manifests using Helm for NGINX Plus.
```

Comment:
We'll also need to put the CRDs and Services into the plus directory, like how OSS does.

Reply:
will do

Reply:
Keeping this thread open, was running into errors with generating CRDs and copying it to Plus folder 

```
deploy/manifests/nginx-plus/crds/gateway.nginx.org_nginxgateways.yaml
  21:11     error    wrong indentation: expected 21but found 10  (indentation)
  45:19     error    wrong indentation: expected 29but found 18  (indentation)
  63:21     error    wrong indentation: expected 32but found 20  (indentation)
  64:21     error    wrong indentation: expected 32but found 20  (indentation)
  65:21     error    wrong indentation: expected 32but found 20  (indentation)
  66:21     error    wrong indentation: expected 32but found 20  (indentation)
  67:21     error    wrong indentation: expected 32but found 20  (indentation)
  68:21     error    wrong indentation: expected 32but found 20  (indentation)
  69:21     error    wrong indentation: expected 32but found 20  (indentation)
  70:21     error    wrong indentation: expected 32but found 20  (indentation)
  71:21     error    wrong indentation: expected 32but found 20  (indentation)
  76:121    error    line too long (162 > 120 characters)  (line-length)
  88:121    error    line too long (162 > 120 characters)  (line-length)
  95:121    error    line too long (124 > 120 characters)  (line-length)
  115:121   error    line too long (139 > 120 characters)  (line-length)
```

will keep digging why that is happening

Reply:
We have an exception for this linter in the existing CRDs directory here: https://github.com/nginxinc/nginx-gateway-fabric/blob/main/.yamllint.yaml#L44

I'm not convinced we need two separate folders though, especially for CRDs, it seems confusing to me. Can't we just have two deployment manifests and keep the rest as common?

Reply:
> Can't we just have two deployment manifests and keep the rest as common?

Yeah, I actually think that makes more sense. It really is only one file that is different. We could have `nginx-gateway.yaml` and `nginx-gateway-plus.yaml`.

Reply:
Update: I continued this thought in https://github.com/nginxinc/nginx-gateway-fabric/pull/1524/files#r1484519978